data/config-schema/src/main/resources/net/corda/schema/configuration/p2p.linkManager/1.0/corda.p2p.linkManager.json

```
@@ -91,10 +91,29 @@
     },
     "sessionsPerPeer": {
       "description": "The number of actively maintained sessions between two peers.",
-      "type": "integer",
-      "default": 4,
+      "type": ["integer", "null"],
+      "default": null,
       "minimum": 1
     },
+    "numOfSessionsPerPeer": {
+      "description": "The number of actively maintained sessions between two peers.",
+      "type": "object",
+      "default": {},
+      "properties": {
+        "forMgm": {
+          "description": "The number of actively maintained sessions between two peers if one of them is an MGM.",
```

Might be nice to give a little more context. For example: 
```suggestion
          "description": "The number of actively maintained sessions between two peers if one of them is an MGM. Communication between an MGM and a member is not considered critical and so the default value of 1 is recommended to improve performance.",
```

---

addressed in [980cb7f](https://github.com/corda/corda-api/pull/1307/commits/980cb7f1e2b40256508cee92a25f223b288ffb15)